SchoolNormalization.md

```diff
@@ -0,0 +1,234 @@
+SchoolNormalizationService
+=============
+
+Table of Contents
+_________
+- [Request Information](#request-information)
+- [Sample Response](#sample-response)
+- [Versioning](#versioning)
+- [Supported Countries](#supported-countries)
+
+
+
+#Request Information
+
+
+HTTP method: GET or POST
+Parameters (query/form):
+-        query (required) : The educational institution information to be queried.
+-        country (optional) : 2 digit ISO country code of educational institution. Accepted country codes are [here](#supported-countries)
+ 
+Example: https://api.careerbuilder.com/core/normalizedschools?query=georgia%20tech&country=us
+
+#Sample Response
+
+
+```
+{
+    "data": {
+        "normalized_schools": [
+            {
+                "normalized_school_name": "Georgia Institute of Technology",
+                "id": "53bff579e4b04710d09fa98d",
+                "country": "US",
+                "confidence": 2
```

Comment:
If we aren't ordering, we should be, and it should be indicated in the documentation.

From: Thomas Mahoney <notifications@github.com<mailto:notifications@github.com>>
Reply-To: cbdr/DataScienceAPIDocumentation <reply@reply.github.com<mailto:reply@reply.github.com>>
Date: Tuesday, September 22, 2015 at 3:04 PM
To: cbdr/DataScienceAPIDocumentation <DataScienceAPIDocumentation@noreply.github.com<mailto:DataScienceAPIDocumentation@noreply.github.com>>
Subject: Re: [DataScienceAPIDocumentation] Add school norm (#2)


In SchoolNormalization.md<https://github.com/cbdr/DataScienceAPIDocumentation/pull/2#discussion_r40129631>:

> +-        country (optional) : 2 digit ISO country code of educational institution. Accepted country codes are [here](#supported-countries)
> +
> +Example: https://api.careerbuilder.com/core/normalizedschools?query=georgia%20tech&country=us
> +
> +#Sample Response
> +
> +
> +```
> +{
> +    "data": {
> +        "normalized_schools": [
> +            {
> +                "normalized_school_name": "Georgia Institute of Technology",
> +                "id": "53bff579e4b04710d09fa98d",
> +                "country": "US",
> +                "confidence": 2


is confidence always an integer? what is the possible range? these are things that IMO aren't clear from the documentation. I think a small section explaining the fields would be good, and/or a different example response that shows multiple schools with varying confidence. if we guarantee results in decreasing order of confidence, we should say that too.

-
Reply to this email directly or view it on GitHub<https://github.com/cbdr/DataScienceAPIDocumentation/pull/2/files#r40129631>.